packages/pds/src/api/app/bsky/feed/getFeed.ts

```diff
@@ -1,10 +1,11 @@
 import { Server } from '../../../../lexicon'
 import AppContext from '../../../../context'
+import { forwardResHeader } from '@atproto/xrpc-server/src/util'
```

Comment:
I wonder if this could create issues at runtime in the build, similar to what we've seen when trying to dip into the `@atproto/api/...`.

Reply:
ah good call - i missed this import

ended up just passing headers through the route return statement